### DISCUSSION TITLE
[Feature] Preserve URL when browsing /photos change URL depending where you are in the timeline

### DISCUSSION THREAD

**Discussion by Sums123** [immich-app/immich#6999] [Feature Request]
### The feature

Often I add new photos and I am in the middle of my timeline. I would like to see the new additions and therefore refresh the page, however I always land at the top of the page. The same happens when I go to a photo element and refresh (e.g. to see if new face data has become available) - I think it would be great if there could be specific URLs depending on where you are

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #7017
